framework/src/mfem/submeshes/MFEMCutTransitionSubMesh.C

```diff
@@ -179,7 +179,7 @@ MFEMCutTransitionSubMesh::labelMesh(mfem::ParMesh & parent_mesh)
 mfem::Vector
 MFEMCutTransitionSubMesh::findFaceNormal(const mfem::ParMesh & mesh, const int & face)
 {
-  MFEM_ASSERT(mesh->SpatialDimension() == 3,
+  MFEM_ASSERT(mesh.SpaceDimension() == 3,
```

Comment:
How did this work before?!

Reply:
Should we do a debug build of MFEM inside the CI containers so that we can use MFEM debug in https://civet.inl.gov/job/3221338/ too?

Just need to do that here https://github.com/idaholab/moose/blob/e605d5e8f6e883fa354b2344665ce057013a0bb5/apptainer/moose-dev.def#L338.

I honestly wouldn't object to requiring in the MOOSE linking that if we're doing moose METHOD=dbg that we require a Debug build of MFEM